controller.go

```diff
@@ -449,7 +454,7 @@ func jobLabels(apprepo *apprepov1alpha1.AppRepository) map[string]string {
 
 // cronJobName returns a unique name for the CronJob managed by an AppRepository
 func cronJobName(apprepo *apprepov1alpha1.AppRepository) string {
-	return fmt.Sprintf("apprepo-%s", apprepo.GetName())
+	return fmt.Sprintf("apprepo-%s-%s", apprepo.GetName(), apprepo.GetUID())
```

Comment:
why do we need to add the UID?

Reply:
it is not clear because this `cronJobName` is used as `generateName` not the usual `name` (which is what I thought at the beginning), let me retry